ui/decredmaterial/line.go

```diff
@@ -13,12 +13,20 @@ import (
 type (
 	// Line represents a rectangle widget with an initial thickness of 1
 	Line struct {
-		Height int
-		Width  int
-		Color  color.NRGBA
+		Height     int
+		Width      int
+		Color      color.NRGBA
+		isVetircal bool
 	}
 )
 
+// VLine returns a vertical line widget instance
+func (t *Theme) VLine(height, width int) Line {
```

Comment:
A better name for this could be "SeparatorVertical" similar to the Separator widget created in line.go